<launch>

    <arg name="uav_id" default="3"/>
    <arg name="autopilot" default="px4"/>

    <node name="agent_sm_node" output="screen" pkg="muav_state_machine" type="agent_sm_node.py" ns = "agent_sm_node" >
        <param name="uav_id" value="$(arg uav_id)" />
        <param name="autopilot" value="$(arg autopilot)" />
    
    
    </node>
    
    

</launch>